docs/quickstarts/sso-quickstart.md

```diff
@@ -119,24 +119,24 @@ Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
 
 Complete the following steps to try out an Outlook add-in.
 
-1. When the SSO configuration process completes, run the following command to build the project and start the local web server.
+1. When the SSO configuration process completes, run the following command to build the project and start the local web server, and sideload your add-in in the previously selected Office client application.
```

Comment:
```suggestion
1. When the SSO configuration process completes, run the following command to build the project and start the local web server, and sideload your add-in in Outlook on Windows.
```

Reply:
There is no sideloading for Outlook yet, so the above string should be updated to not include "sideload" and we should provide the instructions for how a user manually registers an add-in for Outlook